website/source/api/agent/service.html.md

```diff
@@ -151,6 +151,194 @@ contains the [hash-based blocking
 query](/api/index.html#hash-based-blocking-queries) hash for the result. The
 same hash is also present in `X-Consul-ContentHash`.
 
+## Get local service health
+
+The `/v1/agent/health/service/name/<service_name>` and
+`/v1/agent/health/service/id/<service_id>` endpoints allow retrieving an
+aggregated state of service(s) of the local agent.
+
+The endpoint `/v1/agent/health/service/name/<service_name>` queries all
+services with a given names (several may match), while
+`/v1/agent/health/service/id/<service_id>` will match a single service only.
+
+If you know the ID of service you want to target, it is recommended to use
+the version `/v1/agent/health/service/id/<service_id>` so you have the result
+for the service only. When requesting
+`/v1/agent/health/service/name/<service_name>`, the caller will receive the
+worst state of all services having the given name.
+
+Those endpoints support JSON format and text/plain formats, JSON being the
+default. In order to get the text format, you can append `?format=text` to
+the URL or use Mime Content negotiation by specifying a HTTP Header
+`Accept` starting with `text/plain`.
+
+Those endpoints return the aggregated values of all healthchecks for the
+service and will return the corresponding HTTP codes:
+
+| Result | Meaning                                                |
+| ------ | ------------------------------------------------------ |
+| `200`  | All healthchecks of this service are passing           |
```

Comment:
nit: suggest we drop "of this service" or change it to "of every matching service instance" which seems unnecessary.